src/components/tools/edit/EditTool.js

```diff
@@ -140,24 +134,15 @@ function EditToolDialog(props) {
     const handleSubmit = (values, { props }) => {
         const submission = { ...values };
 
-        //these keys needs to excluded from submission to avoid service errors
-        delete submission.implementation;
-        delete submission.permission;
-        delete submission.is_public;
-        delete submission.container.image.id;
-
         //these keys needs to added to submission for interactive tools
         if (submission.interactive) {
-            submission.container.interactive_apps = {};
-            submission.container.interactive_apps.cas_url =
-                config.vice.defaultImage;
-            submission.container.interactive_apps.name =
-                config.vice.defaultName;
-            submission.container.interactive_apps.image =
-                config.vice.defaultCasUrl;
-            submission.container.interactive_apps.cas_validate =
-                config.vice.defaultCasValidate;
-            submission.container.skip_tmp_mount = true;
```

Comment:
`submission.container.skip_tmp_mount = true` is still required.

Reply:
Oops. Thats a boo boo.